test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -375,6 +386,40 @@ public void createParticipantEmailEnabledVerificationWanted() {
         verify(account).setStatus(AccountStatus.UNVERIFIED);
     }
     
+    @Test
+    public void createParticipantWithPhoneEmailEnabledVerificationWanted() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval(null, null);
+
+        StudyParticipant phoneOnly = new StudyParticipant.Builder()
+                .copyOf(PARTICIPANT).withEmail(null).build();
+        
+        participantService.createParticipant(STUDY, CALLER_ROLES, phoneOnly, true);
+        
+        verify(accountWorkflowService, never()).sendEmailVerificationToken(any(), any(), any());
+        // The account will not be verified.
+        verify(account).setStatus(AccountStatus.UNVERIFIED);
+        verify(account).setPhone(PHONE);
+        verify(account).setPhoneVerified(Boolean.FALSE);
+    }
+    
+    @Test
+    public void createParticipantWithPhoneEmailEnabledVerificationNotWanted() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval(null, null);
+
+        StudyParticipant phoneOnly = new StudyParticipant.Builder()
+                .copyOf(PARTICIPANT).withEmail(null).build();
+        
+        participantService.createParticipant(STUDY, CALLER_ROLES, phoneOnly, false);
+        
+        verify(accountWorkflowService, never()).sendEmailVerificationToken(any(), any(), any());
+        // The account will not be verified.
+        verify(account).setStatus(AccountStatus.ENABLED);
+        verify(account).setPhone(PHONE);
+        verify(account).setPhoneVerified(Boolean.FALSE);
```

Comment:
We should also check that setEmailVerified() is not set to true, to make sure that the if-statement is correct.

Reply:
I think I covered this case now in createParticipantEmailEnabledVerificationWanted(). I also verified in a couple of update tests that the user cannot set emailVerified = true through the object they submit (it always starts out as false).